packages/core/src/domain/configuration.ts

```diff
@@ -42,6 +42,7 @@ export interface UserConfiguration {
   publicApiKey?: string // deprecated
   clientToken: string
   applicationId?: string
+  actionNameAttribute?: string
```

Comment:
I'm wondering if the name is not misleading.
It can be confused with the [name of the action object we collect](https://docs.datadoghq.com/real_user_monitoring/browser/data_collected/#action-attributes) 
Maybe something like this is clearer:
- actionNameSelectorAttribute
- actionNameHtmlAttribute
- actionNameTrackingAttribute (to refer to the [tracking section of the doc](https://docs.datadoghq.com/real_user_monitoring/browser/tracking_user_actions/?tab=npm))

Reply:
we will add some doc about it to explain the usage so IMO it would be clearer enough.

@hdelaby @BenoitZugmeyer @webNeat any thoughts?

Reply:
About the wording: `actionNameAttribute` + doc LGTM, it's simple enough.

About the configuration: maybe we could have a more abstract `getActionName(element, event)` function instead, to allow the customer to implement their own logic to add beside our logic? That would be a bit more flexible.

Reply:
a more abstract strategy sounds interesting but I'd be in favor of waiting to have the need, in order to see what could be the most relevant approach to do that.

Reply:
Sounds good to me!

Reply:
Yes, good to me too :)

Reply:
I was thinking that having an array of attributes to check for, instead of just one would offer more flexibility, but the function `getActionName` also is interesting.
The current solution sounds enough to me for now.

Reply:
The function idea is great! Love that concept.